roles/forkliftcontroller/templates/deployment-validation.yml.j2

```diff
@@ -48,11 +48,20 @@ spec:
               value: 'false'
 {% endif %}
           volumeMounts:
+{% if validation_allow_user_supplied_rules|bool %}
+            - name: {{ validation_extra_volume_name }}
+              mountPath: {{ validation_extra_volume_mountpath }}
+{% endif %}
 {% if validation_tls_enabled|bool %}
             - name: {{ validation_tls_secret_name }}
               mountPath: /var/run/secrets/{{ validation_tls_secret_name }}
 {% endif %}
       volumes:
+{% if validation_allow_user_supplied_rules|bool %}
+        - name: {{ validation_extra_volume_name }}
+          configMap:
+            name: {{ validation_configmap_name }}
+{% endif %}}
```

Comment:
Need to remove the extra curly brackets on that endif clause , jinja2 will complain

Reply:
fixed, thanks